<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientFrm</class>
 <widget class="QMainWindow" name="ClientFrm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>971</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientFrm</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Numéro du port</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="serverPort">
              <property name="maximum">
               <number>10000</number>
              </property>
              <property name="value">
               <number>2048</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Adresse du serveur</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="serverAdress">
              <property name="text">
               <string>127.0.0.1</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="connectBtn">
          <property name="text">
           <string>Connexion</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="photoBtn">
          <property name="text">
           <string>Photo</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="imgLbl">
            <property name="text">
             <string>Label image</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="lblImg">
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="videoBtn">
          <property name="text">
           <string>Tracking</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="TotoLabel" name="outputLbl">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>480</height>
           </size>
          </property>
          <property name="text">
           <string>waiting for images</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Messages</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="msgBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Message à envoyer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="msgToSend"/>
        </item>
        <item>
         <widget class="QPushButton" name="sendBtn">
          <property name="text">
           <string>Envoyer au serveur</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="quitBtn">
        <property name="text">
         <string>Quitter</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TotoLabel</class>
   <extends>QLabel</extends>
   <header>totolabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
